project/src/main/java/com/google/sps/servlets/BrowseServlet.java

```diff
@@ -114,8 +114,9 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     searchInfo.setProperty("imageUrl", null);
     searchInfo.setProperty("textSearch", null);
     if (userUploadedImage) {
-      Map<String, List<FileInfo>> files = blobstore.getFileInfos(request);
-      String gcsUrl = CloudStorageLibrary.getGcsFilePath(files);
+      //Map<String, List<FileInfo>> files = blobstore.getFileInfos(request);
+      // String gcsUrl = CloudStorageLibrary.getGcsFilePath(files);
```

Comment:
Do we still need this line? I was just wondering since it doesn't follow the syntax in the Cloud storage library

Reply:
You right